doc/project/checklist.md

```diff
@@ -184,6 +179,20 @@ Any other software-visible design changes have been reviewed by the software tea
 
 All known "Won't Fix" bugs and "Errata" have been reviewed by the software team.
 
+### SEC_CM_IMPLEMENTED
+
+Any appropriate security counter-measures are implemented.
+For redundantly encoded FSMs, [the sparse-fsm-encode.py script](https://github.com/lowRISC/opentitan/blob/master/util/design/sparse-fsm-encode.py) must be used to generate the encoding.
+
+### SEC_NON_RESET_FLOPS
+
+A review of sensitive security-critical storage flops was completed.
+Where appropriate, non-reset flops are used to store secure material.
+
+### SEC_SHADOW_REGS
+
+Shadow registers are implemented for all appropriate storage of critical control functions.
```

Comment:
Minor nit: [Totally fine with keeping this, but] Isn't this _technically_ a counter-measure, and hence covered by `SEC_CM_IMPLEMENTED`?

Reply:
technically you are correct. but I feel like its ok to leave this as a separate item, since this is a standardized countermeasure available in `reggen`. `SEC_CM_IMPLEMENTED` is more meant to cover custom / manually added countermeasures in the body of the design.

Reply:
Going by that argument, aren't we standardizing other countermeasures via `prims`? Either we list ALL countermeasures as dedicated checklist items or none - singling one a few seems arbitrary.

Reply:
I agree with @sriyerg here. This seems like something that belongs on the much more specific checklist for "are we happy with the countermeasures for this block?", rather than "is the block written yet?".

Reply:
Giving an exhaustive list of countermeasures is going to be difficult due to the custom nature of some of the countermeasures. Some are easier to standardize (e.g. FSM state encoding, counters, lfsrs, for which we can make prims). Others are entirely custom for the data path at hand (AES, KMAC, CFI etc). So lumping them together under the same checklist item probably makes more sense.

As for `SEC_SHADOW_REGS`, I am ok with moving that under the `SEC_CM_*` if you feel that would be cleaner . I was just pointing out that it is the only one that is completely automated via reggen, and because of that we may consider leaving it separate.

Regarding @rswarbrick's point, should we change the last checklist item on countermeasures to something like `SEC_CM_IMPL_AND_REVIEWED`? or should we even create a new item called `SEC_CM_REVIEWED` in order to address that point explicitly?